modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -8,6 +8,8 @@
 
 You can use the installed {product-title} Prometheus monitoring as a source for the metrics used by the custom metrics autoscaler. However, there are some additional configurations you must perform.
 
+For your scaled objects to be able to read the {product-title} Prometheus metrics, you must use a trigger authentication or a cluster trigger authentication in order to provide the authentication information required. The following procedure differs depending on which trigger authentication method you use. For more information on trigger authentications, see "Understanding custom metrics autoscaler trigger authentications". 
```

Comment:
can you add somewhere before the procedure that it is using the trigger authentication method? unless it's not, but based on the information contained in some of the codeblocks within the procedure that seems to be the case

Reply:
Made the procedure generic, for both auth methods.